deployment/helpers/solana.ts

```diff
@@ -140,16 +150,37 @@ export async function ledgerSignAndSend(connection: Connection, instructions: Tr
 
   tx.add(...instructions);
 
-  const recentBlockHash = await connection.getLatestBlockhash();
+  if (priorityFeeConfig) {
+    const priorityFee = await getPrioritizationFee(connection, 
+      priorityFeeConfig.lockedWritableAccounts, 
+      priorityFeeConfig.priorityFeePolicy, 
+      priorityFeeConfig.defaultFee);
+    console.log(`Using priority fee policy: ${priorityFeeConfig.priorityFeePolicy}, selected fee: ${priorityFee} microLamports`);
+    const addPriorityFeeIx = ComputeBudgetProgram.setComputeUnitPrice({ 
+      microLamports: priorityFee
+    });
+    tx.add(addPriorityFeeIx);
+  }
 
-  tx.recentBlockhash = recentBlockHash.blockhash;
```

Comment:
This shouldn't be removed. The recent block hash passed into the confirm tx function should be the same as the one that was signed.

Reply:
Note that it was moved to a later step but it alters the semantics.

Reply:
Please recheck: now the sendRaw...and confirm... are perfomed using the same blockhash assigned in the tx field upfront..